lib/matplotlib/lines.py

```diff
@@ -1031,6 +1055,25 @@ def set_color(self, color):
         self._color = color
         self.stale = True
 
+    def set_offcolor(self, offcolor):
+        """
+        Set the gap color for the dashed line style.
+
+        .. warning::
+
+            In some cases the two line colors may overlap, which could lead to
+            unwanted effects if also setting ``alpha``.
```

Comment:
The multicolored line is somewhat different from dashes because it assigns values/borders in data coordinates. I.e. you can zoom in to one color segment. With dashes however, we don't want the users to zoom in to a single dash. The dash size is independed of the data coordiantes.

Anyway, I've proposed to decleare this feature experimental so that we still have the freedom to come up with a different implementation.